docs/assets/styles/determined.css

```diff
@@ -98,8 +100,39 @@ html[data-theme="dark"] {
   will-change: transform;
   /* layout */
   box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
+  display: grid;
```

Comment:
switch to grid layout since flex layout cause some layout issue

#### Before

<img width="706" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/fcbd0012-3926-46f4-81ea-fc702f6e7b17">


#### After

<img width="650" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/f2a63e6c-bfb4-4473-a246-37e553163fb3">